### PR TITLE
[4.0] CS for Drone

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes

Remove tabs.

### Testing Instructions

Inspect Drone/phpcs.

### Expected result

No errors.

### Actual result
```
FILE: ...rone/src/administrator/components/com_finder/Indexer/Indexer.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
316 | ERROR | [x] Whitespace found at end of line
| | (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine) 
```
### Documentation Changes Required

No.

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 23c549eddca56650cac4ddf9b18f91001b846b75<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26346">issues.joomla.org/tracker/joomla-cms/26346</a>.</sub>

File: administrator/components/com_finder/Indexer/Indexer.php [hunk 1]
```diff
@@ -313,7 +313,7 @@ public function remove($linkId)
 			->delete($db->quoteName('#__finder_links'))
 			->where($db->quoteName('link_id') . ' = :linkid')
 			->bind(':linkid', $linkId, ParameterType::INTEGER);
-		
+
```
Comment (Quy) [inline administrator/components/com_finder/Indexer/Indexer.php]:
```suggestion
```

Comment (Quy) [reply administrator/components/com_finder/Indexer/Indexer.php]: See above code where there is no blank line.

Comment (SharkyKZ) [reply administrator/components/com_finder/Indexer/Indexer.php]: It's fine either way but OK.

Comment (Quy) [reply administrator/components/com_finder/Indexer/Indexer.php]: Yes, but mainly for consistency.

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on b772fc353560799466097a99492092c898232cd6<br /><br />review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26346">issues.joomla.org/tracker/joomla-cms/26346</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b772fc353560799466097a99492092c898232cd6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26346">issues.joomla.org/tracker/joomla-cms/26346</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26346">issues.joomla.org/tracker/joomla-cms/26346</a>.</sub>

**Review by richard67 — APPROVED** [review]

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**